controllers/propagator/replication_utils.go

```diff
@@ -16,7 +16,7 @@ import (
 )
 
 // labelsForRootPolicy returns the labels for given policy
-func labelsForRootPolicy(plc *policiesv1.Policy) map[string]string {
+func LabelsForRootPolicy(plc *policiesv1.Policy) map[string]string {
```

Comment:
https://github.com/open-cluster-management-io/governance-policy-propagator/pull/58

@JustinKuli @mprahl  I know neither this way nor making some similar code in `policyautomation_controller.go` looks nice, let me know your preference.

Reply:
This is fine from my perspective.

Reply:
Yeah, seems fine to me as well